domain/credentials/service.go

```diff
@@ -46,28 +46,27 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
 		if customer.City == nil {
 			return fmt.Errorf("customer.City must be set for issuing a credential")
 		}
-		issue := true
-		for _, credential := range credentials {
-			if credential.Organization.Name == customer.Name && credential.Organization.City == *customer.City {
-				// Customer already has credential with given name/ & city, so we don't need to issue one.
-				issue = false
-				break
+
```

Comment:
Indenting is weird?

Reply:
It's not? Note that I'm running `gofmt` on every file.